newspack-ads.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Plugin Name:     Newspack Gds
+ * Plugin URI:      https://newspack.blog
+ * Description:     Ad services integration.
+ * Author:          Automattic
+ * License:         GPL2
+ *
+ * @package         Newspack
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+// Define NEWSPACK_GAM_PLUGIN_FILE.
```

Comment:
GAM remnant

Reply:
Fixed in https://github.com/Automattic/newspack-ads/pull/4/commits/dbfbcbc47b76636d74d4dbb26a38dead7cc78641